<launch>
  <!-- Basic launch file for one robot in the match_world_common map-->

  <include file="$(find match_gazebo)/launch/scale.launch">
  </include>

  <arg name="robot1_name"  default="mir3"/>
  
  <!-- Choose group for creating a namespace environment if you want to add multiple nodes/includes to that namespace easily -->
  <group ns="$(arg robot1_name)">
    <include file="$(find mir_launch_sim)/launch/mir_200.launch">
        <arg name="robot_x"     default="26.0" />
        <arg name="robot_y"     default="26.0" />
        <arg name="robot_yaw"   default="0.0" /> 
        <arg name="tf_prefix"   default="$(arg robot1_name)" />       
        <arg name="model_name"  default="$(arg robot1_name)"/>
    </include>
  </group> 

  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find mir_examples)/rviz/formation_path_planning_visualization.rviz" /><!--2D Navigation-->
</launch>